custom/inddex/filters.py

```diff
@@ -0,0 +1,197 @@
+from django.utils.translation import ugettext as _
+
+from corehq.apps.reports.filters.base import BaseSingleOptionFilter
+from custom.inddex.sqldata import FoodCodeData, FoodBaseTermData
+from corehq.apps.reports.filters.dates import DatespanFilter
+
+
+class DateRangeFilter(DatespanFilter):
+    label = _('Date Range')
```

Comment:
You can't use `ugettext` on module level variables.  You probably want `ugettext_lazy` here.
https://commcare-hq.readthedocs.io/translations.html